### PR TITLE
Add ability to change period of time metrics are shown for.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**

### CODE REVIEW

File: public/constants/timeframes.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const timeframes = [
+    { queryParamValue: "last-week", prettyName: "Last week" },
```
Comment (LATaylor-guardian) [inline public/constants/timeframes.js]: Not sure how helpful last week on it's own is - will probably remove.

File: public/services/TimeframeService.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        }
+
+        const matchedTimeframe = timeframes.filter( timeframe => {
+            return timeframeValue === timeframe.queryParamValue
+        });
+        
+        const timeframe = R.defaultTo('', R.head(matchedTimeframe));
+
+        let decideStartDate = R.cond([
+            [R.equals("last-week"),        R.always(formattedDate(1))],
```
Comment (LATaylor-guardian) [inline public/services/TimeframeService.js]: Again will probably remove as not sure how helpful one week is.

Comment (oilnam) [inline public/services/TimeframeService.js]: I second this

**Comment by oilnam** [comment]
👍

**Pull request merged**